docker-compose.override.yml

```diff
@@ -1,100 +1,33 @@
 
-version: '2.1'
-
-#volumes:
-#  emberosf-sync:
-#    external: true
+version: "3"
 
 services:
-
-#  fakecas:
-#    image: quay.io/centerforopenscience/fakecas:latest
-#    command: fakecas -host=0.0.0.0:8080 -dbaddress=172.19.0.7:27017
-
-#  wb:
-#    volumes_from:
-#      - container:wb-sync
-
-#  wb_worker:
-#    volumes_from:
-#      - container:wb-sync
-
-#  wb_requirements:
-#    volumes_from:
-#      - container:wb-sync
-
-#  mfr:
-#    volumes_from:
-#      - container:mfr-sync
-
-#  mfr_requirements:
-#    volumes_from:
-#      - container:mfr-sync
-
-#  preprints:
-#    volumes_from:
-#      - container:preprints-sync
-#########    Use this for ember-osf linked development (with docker-sync): #############
```

Comment:
We should still included commented blocks to enable local development for ember-osf, waterbutler, mfr, etc.